server/authentication/role.ts

```diff
@@ -0,0 +1,5 @@
+const enum Role {
```

Comment:
Did some reading around this, apparently not good practice to export **const enum**s in Typescript. Article here: https://ncjamieson.com/dont-export-const-enums/

Not too great at my Typescript to know whether we care about this. But I thought I'd point it out in case anyone had an alternative?

Also not too happy with the location of this file, might be better suited in a /types folder? Any thoughts?